doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -92,9 +92,10 @@ A single insert is composed for each notified entity, containing such insert the
 * `fiwareServicePath`: Notified fiware-servicePath, or the default configured one if not notified.
 * `entityId`: Notified entity identifier.
 * `entityType`: Notified entity type.
-* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, it can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
-    * Either of type `geo:point`.
-    * Either having associated a `location` metadata of type `string` and value `WGS84`.
+* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, but it must only contain points. It can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
+    * Either of type `geo:point` (a point).
+    * Either of type `geo:json` (a GeoJson representing a point).
+    * Either having associated a `location` metadata of type `string` and value `WGS84`. Please observe this option is deprecated.
```

Comment:
Maybe "this option is deprecated" could be a link to https://fiware-orion.readthedocs.io/en/master/deprecated/index.html